CMakeLists.txt

```diff
@@ -109,10 +109,40 @@ install(TARGETS minizip EXPORT minizip-exports
   LIBRARY DESTINATION "lib"
   ARCHIVE DESTINATION "lib")
 
+set(INSTALL_CMAKE_DIR "cmake")
 install(EXPORT minizip-exports
-        DESTINATION "cmake"
+        DESTINATION "${INSTALL_CMAKE_DIR}"
         NAMESPACE "MINIZIP::")
 
+# Create and install CMake package config version file to allow find_package()
+
+include(CMakePackageConfigHelpers)
+write_basic_package_version_file(
+  "${CMAKE_CURRENT_BINARY_DIR}/minizipConfigVersion.cmake"
+  VERSION 1.2
+  COMPATIBILITY SameMajorVersion)
+
+# Create config for find_package()
+
+file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/minizipConfig.cmake.in"
+"
+@PACKAGE_INIT@
+include(CMakeFindDependencyMacro)
```

Comment:
What is this? And does the master branch need this chunk of cmake?

Reply:
Without the `find_dependency` function the consuming CMake project has to call `find_package(ZLIB)` in order to use minizip

```cmake
find_package(ZLIB)
find_package(minizip)
target_link_libraries(myapp PRIVATE MINIZIP::minizip)
```

With the `find_dependency` function the consuming project just needs to find the minizip package.

```cmake
find_package(minizip)
target_link_libraries(myapp PRIVATE MINIZIP::minizip)
```

That way someone using just minizip doesn't need to know about the fact, that it's internally using ZLIB.

Additionally, since ZLIB is a mandatory dependency for minizip the `find_dependency` function also ensures, that minizip is reported as "not found", if ZLIB wasn't found.

I think, it would be good, if the master branch also contains this chunk of cmake.